ci-operator/step-registry/ipi/gcp/private/post/ipi-gcp-private-post-chain.yaml

```diff
@@ -0,0 +1,9 @@
+chain:
```

Comment:
we already added flatten steps in ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/private/cucushift-installer-rehearse-gcp-ipi-private-workflow.yaml, we do not need this file.